src/ft2font_wrapper.cpp

```diff
@@ -1547,11 +1519,13 @@ PyMODINIT_FUNC PyInit_ft2font(void)
     FT_Library_Version(_ft2Library, &major, &minor, &patch);
     snprintf(version_string, sizeof(version_string), "%d.%d.%d", major, minor, patch);
 
-    PyObject *m = PyModule_Create(&moduledef);
-    if (!m ||
-        !PyFT2Image_init_type(m, &PyFT2ImageType) ||
-        !PyGlyph_init_type(m, &PyGlyphType) ||
-        !PyFT2Font_init_type(m, &PyFT2FontType) ||
+    PyObject *m;
+    if (!(m = PyModule_Create(&moduledef)) ||
+        prepare_and_add_type(PyFT2Image_init_type(), m) ||
+        prepare_and_add_type(PyFT2Font_init_type(), m) ||
+        // Glyph is not constructible from Python, thus not added to the module.
+        PyType_Ready(PyGlyph_init_type()) ||
+        PyModule_AddObject(m, "FT2Font", (PyObject*)&PyFT2FontType) ||
```

Comment:
Is this not done on line 1525's `prepare_and_add_type`?

Reply:
oops, fixed.